src/ex4/dist/app.js

```diff
@@ -0,0 +1,102 @@
+const addButton = document.getElementById("list-item-submit")
+const textInput = document.getElementById("list-item-input")
+const allLists = document.getElementById("list")
+const counterLists = document.getElementById("counter")
+const errorMessage = document.getElementById("error")
+const regex = /^[0-9]*$/
+
+const itemC = new ItemClient()
+
+class Main {
+  constructor() {
+    this.name = "Main"
+  }
+  init() {
+    addButton.addEventListener("click", function () {
+      validation()
+    })
+  }
+}
+function addNewList(name) {
+  const li = document.createElement("li")
+  const hr = document.createElement("hr")
+  const deleteButton = document.createElement("button")
+  deleteButton.setAttribute("class", "dltbtn")
+  deleteButton.setAttribute("id", textInput.value)
+  deleteButton.innerHTML = "X"
+
+  deleteButton.addEventListener("click", async function (e) {
+    li.remove()
+    hr.remove()
+    await itemC.deleteItem(e.target.id)
```

Comment:
this doesn't seem to work..
![image](https://user-images.githubusercontent.com/36815567/174827128-d4a925aa-ff4f-4e68-97d2-63e2885b85ad.png)
![image](https://user-images.githubusercontent.com/36815567/174827239-aad3b060-025e-4f72-af92-0e62b0e55bec.png)
![image](https://user-images.githubusercontent.com/36815567/174827281-bea5633e-9743-4a98-a761-3bff91397435.png)